docs/guide/tasks/multilabel_classification_task.md

```diff
@@ -0,0 +1,132 @@
+        "output_guidelines": "You will return the answer as a comma separated list of labels sorted in alphabetical order. For example: \"label1, label2, label3\"",
+        "labels": [
+            "neutral",
+            "anger",
+            "anticipation",
+            ...
+        ],
+        "example_template": "Input: {example}\nOutput: {labels}"
+    }
+}
+```
+
+The `task_type` sets up the config for a specific task, multilabel_classification in this case.
+
+Take a look at the prompt section of the config. This defines the settings related to defining the task and the machinery around it.
+
+The `task_guidelines` key is the most important key, it defines the task for the LLM to understand and execute on. In this case, we first set up the task and tell the model the kind of data present in the dataset, by telling it that it is an expert at classifying tweets. Next, we define the task more concretely using labels appropriately. `{labels}` will be translated to be all the labels in the `labels` list separated by a newline. These are essential for setting up classification tasks by telling it the labels that it is constrained to, along with any meaning associated with a label.
+
+The `labels` key defines the list of possible labels for the twitter-emotion-detection dataset which is a list of 12 possible labels.
+
+The `example_template` is one of the most important keys to set for a task. This defines the format of every example that will be sent to the LLM. This creates a prompt using the columns from the input dataset, and sends this prompt to the LLM hoping for the llm to generate the column defined under the `label_column`, which is labels in our case. For every input, the model will be given the example with all the columns from the datapoint filled in according to the specification in the `example_template`. The `label_column` will be empty, and the LLM will generate the labels. The `example_template` will be used to format all seed examples.
+
+### Few Shot Config
+
+Let's assume we have access to a dataset of labeled seed examples. Here is a config which details how to use it.
+
+```json
+config = {
+    "task_name": "EmotionClassification",
+    "task_type": "multilabel_classification",
+    "dataset": {
+        "label_column": "labels",
+        "label_separator": ", ",
+        "delimiter": ","
+    },
+    "model": {
+        "provider": "openai",
+        "name": "gpt-3.5-turbo"
+    },
+    "prompt": {
+        "task_guidelines": "You are an expert at classifying tweets as neutral or one or more of the given emotions that best represent the mental state of the poster.\nYour job is to correctly label the provided input example into one or more of the following categories:\n{labels}",
+        "output_guidelines": "You will return the answer as a comma separated list of labels sorted in alphabetical order. For example: \"label1, label2, label3\"",
```

Comment:
Are we still separating the multiple labels by commas or semi-colon?

Reply:
semi-colon is the default but here I used commas and specified it in the config